src/opm/output/eclipse/AggregateActionxData.cpp

```diff
@@ -420,6 +422,18 @@ const std::map<cmp_enum, int> cmpToIndex = {
                     throw std::invalid_argument("Actionx: " + actx.name());
                 }
 
+                /* item[15] is a parameter that indicates whether left_paren or right_paren is used in an expression
+                 * = 0  : no open_paren or left_paren, or both open_paren and right_paren
+                 * = 1  : left_paren at start of condition
+                 * = 2  : right_paren at end of condition
+                */
+
+                if ((cond_it->left_paren) && (!cond_it->right_paren)) {
```

Comment:
Since this is a quite heavy read I would split in two loops - one short to assign `IAcn[ind + 15]` and a second one to assign `iAcn[ind + 17]`